python/orchestrate/_internal/api.py

```diff
@@ -144,13 +144,13 @@ def _get_pagination_parameters(
     return parameters
 
 
-def _get_id_dependent_route(
-    route: str,
+def _get_id_dependent_parameters(
+    id_name: str,
     id_: Optional[str] = None,
-) -> str:
+) -> dict[str, Optional[str]]:
```

Comment:
shouldn't this be returning `dict[str, str]`? We're not setting empty values if parameters are missing.